packages/core/src/readers/SimpleDirectoryReader.ts

```diff
@@ -37,10 +35,10 @@ export class TextFileReader implements BaseReader {
 
 export const FILE_EXT_TO_READER: Record<string, BaseReader> = {
   txt: new TextFileReader(),
-  pdf: new PDFReader(),
+  // pdf: new PDFReader(),
```

Comment:
@himself65 I remember you removed the pdf.js based implementation - as far as I recall the reason was the bundle size, correct? But as pdf.js is working in the browser - should be possible to run it on edge runtime right?

Reply:
Pdfjs is too large not possible in edge runtime